Automattic/jetpack
# Social: Add Threads to social previews

## Proposed changes:

* Add Threads to Social Previews
* Fix Facebook Icon color, a regression introduced in #37977

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:







* Sandbox D152810-code and follow the instructions given there
* Point your site to your sandbox
* Goto post editor
* Open Social/Jetpack sidebar
* Click on Preview
* Confirm that you see Threads tab in preview modal
* Confirm that the preview works as expected in different scenarios
* Confirm that https://github.com/Automattic/wp-calypso/pull/92056 works as expected

<img width="936" alt="image" src="https://github.com/Automattic/jetpack/assets/18226415/4206ca61-4e16-49bb-9c15-79788159e1b9">
